python/felis/datamodel.py

```diff
@@ -255,51 +280,50 @@ def check_units(cls, values: dict[str, Any]) -> dict[str, Any]:
 
         return values
 
-    @model_validator(mode="after")  # type: ignore[arg-type]
-    @classmethod
-    def validate_datatypes(cls, col: Column, info: ValidationInfo) -> Column:
+    @model_validator(mode="after")
```

Comment:
The changes here are correcting the signature and method body of the validation function. I decided to include this in the PR as a minor fix, though it is unrelated to the ticket.